pages/cart/cart.css

```diff
@@ -161,14 +163,117 @@ body {
   color: var(--pink-color);
 }
 
+.address {
+  padding: 0.6rem;
+  background-color: var(--white);
+  margin: 0 0 1rem 0;
+  width: 90%;
+  display: flex;
+  align-items: flex-start;
+}
+
+.address-title {
+  font-size: 1rem;
+  font-weight: bold;
+}
+
+.detail-address {
+  font-size: 1rem;
+}
+
+.click-btn {
+  margin-left: auto;
+  padding: 0.1rem 0.7rem;
+  color: var(--pink-color);
+  border-color: var(--pink-color);
+  background: var(--white);
+}
+
+.click-btn:hover {
+  background: var(--pink-color);
+  color: var(--white);
+}
+
+.address-sec {
+  width: 70%;
+}
+
+.modal{
+  display: flex;
+  align-items: flex-start;
+  justify-content: flex-start;
+  overflow: hidden;
+  display: none;
+}
+
+.modal-content{
+  background-color: var(--product-bgcolor);
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  width: 40%;
+  height: auto;
+  border-radius: 5px;
+}
+
+.input{
+  width: 20rem;
+  margin: 0.5rem 0;
+}
+
+.input-address{
+  height: 7rem;
+  width: 20rem;
+}
+
+.name-label{
+  font-size: 1.1rem;
+  color: var(--bg-color);
+  font-weight: 500;
+  margin-bottom: 1rem;
+}
+
+.cancel-btn{
+  background-color: var(--product-bgcolor);
+  color: var(--bg-color);
+  border-radius: 5xp;
```

Comment:
use rem for border radius

Reply:
I do not t use rems for a value less than 10px.